server/sonar-web/src/main/js/app/components/nav/global/GlobalNavUser.tsx

```diff
@@ -116,27 +113,25 @@ export default class GlobalNavUser extends React.PureComponent {
     return Promise.resolve();
   };
 
-  renderAuthenticated() {
-    const { currentUser, organizations } = this.props;
+  renderAuthenticated(user: LoggedInUser) {
+    const { organizations } = this.props;
     const hasOrganizations = this.props.appState.organizationsEnabled && organizations.length > 0;
     return (
       <li
         className={classNames('dropdown js-user-authenticated', { open: this.state.open })}
         ref={node => (this.node = node)}>
         <a className="dropdown-toggle navbar-avatar" href="#" onClick={this.toggleDropdown}>
-          <Avatar hash={currentUser.avatar} name={currentUser.name} size={24} />
+          <Avatar hash={user.avatar} name={user.name!} size={24} />
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Forbidden non null assertion [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS2966)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary not-null assertion. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS4325)